.github/workflows/python-package.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       matrix:
         python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
         os: [ubuntu-latest, macos-latest, windows-latest]
+        trio: [true, false]
```

Comment:
This is not how I meant by adding trio matrix. This implementation will double the testing time, and if we have more optional dev packages, the testing repeats would grow exponentially.

Ideally, there should be a switch to turn a certain package on/off on specific tests, not all of them. I'll work on that in the future. At mean time, could you remove it from the matrix, and just do trio on Linux/Mac, not Windows, or vice versa, to get some coverage? Like ``orjson``.

Reply:
i set it to run on not-windows since I am testing locally on windows